item-impl/src/main/java/com/example/auction/item/impl/ItemRepository.java

```diff
@@ -168,8 +168,10 @@ private void registerCodec(Session session, TypeCodec<?> codec) {
         }
 
         private CompletionStage<Done> prepareStatements() {
-            session.underlying().thenAccept(s -> registerCodec(s, new EnumNameCodec<>(ItemStatus.class)));
             return doAll(
+                    session.underlying()
+                            .thenAccept(s -> registerCodec(s, new EnumNameCodec<>(ItemStatus.class)))
+                            .thenApply(x -> Done.getInstance()),
```

Comment:
You could also use the `accept` helper from `CompletionStageUtils` that is used elsewhere in this class. It wraps a consumer into a function that returns done. Just makes things a little tidier 😄